docs/howto/how-to-contribute.md

```diff
@@ -34,7 +34,7 @@ This section explains the steps to create a pull request (PR).
 1. Create a commit
 
    It is time to create a commit for submission once you are convinced that your contribution is
-   ready to go. Please include signed-off message at the end of commit message. If not, your pull
+   ready to go. Please include [signed-off message](https://github.com/Samsung/ONE/wiki/ONE-Developer's-Certificate-of-Origin) at the end of commit message. If not, your pull
```

Comment:
How about begin a new line at `at the end...` not to make this line too long?

Reply:
good idea :+1: